### PR TITLE
com.virtualmaker.buildalon 1.0.0

### PR DESCRIPTION
**Pull request by StephenHodgson**
- Initial Release! 🎉🚀

### CODE REVIEW

File: Utilities.BuildPipeline/Packages/com.utilities.buildpipeline/LICENSE.md [hunk 1]
```diff
@@ -1,5 +1,6 @@
 MIT License
 
+Copyright (c) 2024 Buildalon: Automate Unity
```
Comment (afarchy) [inline Utilities.BuildPipeline/Packages/com.utilities.buildpipeline/LICENSE.md:3]:
```suggestion
Copyright (c) 2024 Virtual Maker Corporation
```

File: Utilities.BuildPipeline/Packages/com.utilities.buildpipeline/package.json [hunk 2]
```diff
@@ -1,9 +1,9 @@
 {
-  "name": "com.utilities.buildpipeline",
-  "displayName": "Utilities.BuildPipeline",
-  "description": "The Build Pipeline Utilities aims to give developers more tools and options when making builds with the command line or with continuous integration.",
+  "name": "com.virtualmaker.buildalon",
+  "displayName": "Buildalon",
+  "description": "Buildalon is a comprehensive suite of build, test, and deploy automation solutions for Unity developers.",
```
Comment (afarchy) [inline Utilities.BuildPipeline/Packages/com.utilities.buildpipeline/package.json:4]:
```suggestion
  "description": "Buildalon is a comprehensive suite of automation tools to help Unity developers build, test, and deploy their projects faster.",
```

**Review by afarchy — APPROVED** [review]

File: Utilities.BuildPipeline/Packages/com.utilities.buildpipeline/LICENSE.md (hunk 1 above)
Comment (StephenHodgson) [reply Utilities.BuildPipeline/Packages/com.utilities.buildpipeline/LICENSE.md:3]: This matches with the LICENSE of all the other repos. I can update them all, but I think GitHub defaults it to the name of the Organization.

File: Utilities.BuildPipeline/Packages/com.utilities.buildpipeline/package.json (hunk 2 above)
Comment (StephenHodgson) [reply Utilities.BuildPipeline/Packages/com.utilities.buildpipeline/package.json:4]: This is the same as the tag line for the org. Do you want me to update it there as well?

File: Utilities.BuildPipeline/Packages/com.utilities.buildpipeline/LICENSE.md (hunk 1 above)
Comment (afarchy) [reply Utilities.BuildPipeline/Packages/com.utilities.buildpipeline/LICENSE.md:3]: Yes, copyright needs to belong to a real world entity (person or company). A product can’t own things.

**Pull request merged**